index.ts

```diff
@@ -1,15 +1,14 @@
+const SERVER_URL = "http://localhost:3000/events";
+
 document.addEventListener("DOMContentLoaded", () => {
-  let currentDate: Date = new Date();
-  const dateInLogo = <HTMLElement>document.getElementById("logoText");
-  if (dateInLogo) {
-    dateInLogo.innerText = currentDate.getDate().toString();
+  let currentDate = new Date();
+  const headerIconDate = document.getElementById("logoText");
```

Comment:
```suggestion
  const headerDateIcon = document.getElementById("logoText");
```

almost perfect. with your name i would expect that was to store Date, not be element itself